### ISSUE TITLE
App breaks on wrong NETWORK_ID (REACT_APP_ENVIRONMENT_ID)

### ISSUE THREAD

**Issue by jurajpiar** [rnsdomains/rns-manager-react#269]
This happens when I restart the network and the network id is no longer valid. I change the id, restart the app and all is fine until I login to my wallet.

![Screenshot 2020-04-29 at 20 37 59](https://user-images.githubusercontent.com/36888576/80639022-528e4780-8a59-11ea-8903-6f52579f0ff3.png)

**Comment by jurajpiar** [comment]
It turns out I had to redeploy the RNSSuite after restarting the node.

**Comment by jessgusclark** [comment]
@jurajpiar I'm glad you figured it out!

If you need, I wrote a gist to pull the values from a local ganache network here: https://gist.github.com/jessgusclark/08c728d7e5aac1eba7906d3dd8de1679

**Comment by jurajpiar** [comment]
@jessgusclark , I don't think this should have been closed tho. The fact that I had a wrong env_id does not change the fact that this error should have been caught and this horrible error message should not have replaced the ui.

**Comment by jessgusclark** [comment]
Okay. I'll reopen and update the title of the issue if that is alright.
